gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -296,7 +314,7 @@ synchronized public void stopDag(URI uri) throws IOException {
   private void killFlow(String flowGroup, String flowName, long flowExecutionId) throws IOException {
     int queueId =  DagManagerUtils.getDagQueueId(flowExecutionId, this.numThreads);
     String dagId = DagManagerUtils.generateDagId(flowGroup, flowName, flowExecutionId);
```

Comment:
If we have a dagId static class, it seems to make sense to me that we have a toString() functionality for this and we can remove `DagManagerUtils.generateDagId()`